arbeitszeit_web/www/controllers/request_user_password_reset_controller.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+
+from arbeitszeit.repositories import DatabaseGateway
+
+
+@dataclass
+class RequestUserPasswordResetController:
+    database: DatabaseGateway
+
+    def process_user_password_reset_request(self):
+        raise NotImplemented("not impled")
```

Comment:
"Not implemented."  Actually, does the exception itself print a default message?  If so, you don't need a string at all.